### PR TITLE
Fix synaptic time constants in model documentation

### LINKED ISSUE #1527: Exponential decay time constant in AIF models
Dear All,

I am an MS.c. student of Computational Neuroscience and In my master thesis, I need to change the synaptic decay time constant. As I take a look at the current models, for instance in the "iaf_cond_exp" model, there's no way I could change the synaptic decay time constant. The question is, is there such a model exist so I can tune that parameter, or I should create a new model?

Best regards,
Nosratullah

Note: Sorry to ask this question on the Github page. If there's an active community or forum, I would love to let me know.

### PR DESCRIPTION
**Pull request by clinssen**
Thanks to @nosratullah for pointing this out!

Fixes #1527.

### CODE REVIEW

File: models/aeif_cond_exp.h [hunk 1]
```diff
@@ -237,8 +237,8 @@ class aeif_cond_exp : public Archiving_Node
     double b;          //!< Spike-triggered adaptation in pA
     double V_th;       //!< Spike threshold in mV.
     double t_ref;      //!< Refractory period in ms.
-    double tau_syn_ex; //!< Excitatory synaptic rise time.
-    double tau_syn_in; //!< Excitatory synaptic rise time.
+    double tau_syn_ex; //!< Excitatory synaptic kernel decay time in ms.
+    double tau_syn_in; //!< Inhibitory synaptic kernel decay time in ms.
     double I_e;        //!< Intrinsic current in pA.
 
     double gsl_error_tol; //!< error bound for GSL integrator
```
Comment (sarakonradi) [inline models/aeif_cond_exp.h:244]: I suggest removing these periods for consistency.

File: models/aeif_psc_exp.h [hunk 2]
```diff
@@ -228,8 +228,8 @@ class aeif_psc_exp : public Archiving_Node
     double b;          //!< Spike-triggered adaptation in pA
     double V_th;       //!< Spike threshold in mV.
     double t_ref;      //!< Refractory period in ms.
-    double tau_syn_ex; //!< Excitatory synaptic rise time.
-    double tau_syn_in; //!< Excitatory synaptic rise time.
+    double tau_syn_ex; //!< Excitatory synaptic kernel decay time in ms.
+    double tau_syn_in; //!< Inhibitory synaptic kernel decay time in ms.
     double I_e;        //!< Intrinsic current in pA.
```
Comment (sarakonradi) [inline models/aeif_psc_exp.h:233]: Here too.

**Comment by clinssen** [comment]
f8c67e2 makes the formatting more uniform of these "//!<" type comments, for the parameters struct in all models. I should probably stop at this point before going through the whole NEST codebase.

**Comment by sarakonradi** [comment]
After discussing this offline with @terhorstd, we agreed that #1530 should be merged into master first, as that also includes a clean-up of the model documentation.

**Comment by clinssen** [comment]
@hakonsbm: #1530 is now merged, so review on this PR can begin. Thanks in advance!

**Comment by sarakonradi** [comment]
> @hakonsbm: #1530 is now merged, so review on this PR can begin. Thanks in advance!

Yes, this PR is ready for review! Just removed the "Blocked" label to avoid confusion.

**Pull request merged**